### PR TITLE
Update broken source_code_examples link

### PR DESCRIPTION
**Pull request by micklemacklemore**
Somebody added a link to this page to one of my old repos which has long been deleted! 

I propose updating the link with another PSX example with writing C code. Thanks!

### CODE REVIEW

File: docs/source_code_examples.md [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
 [https://github.com/johanfredin/psx-dev](https://github.com/johanfredin/psx-dev) -  more Code Examples.
 
-[https://github.com/micklemacklemore/mikesPlaystationStuff](https://github.com/micklemacklemore/mikesPlaystationStuff) -  I'm learning to make Playstation 1 games with C
+[https://github.com/micklemacklemore/playstationAssembly](https://github.com/micklemacklemore/psx_cube_demo) -  I'm learning to make Playstation 1 games with C
```
Comment (nicolasnoble) [inline docs/source_code_examples.md]: The preview between the square brackets, and the actual URL between the parenthesis aren't the same URL. Is this intentional? Did you mean to have two different entries there instead?

Comment (micklemacklemore) [reply docs/source_code_examples.md]: My bad, yes I was meant to add two entries there: 

https://github.com/micklemacklemore/playstationAssembly    - An assembly example / tutorial

https://github.com/micklemacklemore/mikesPlaystationStuff     - A C example

I can make another PR request and redo the commit?

Comment (nicolasnoble) [reply docs/source_code_examples.md]: You can just amend this one PR, that's totally fine :)

Comment (micklemacklemore) [reply docs/source_code_examples.md]: Done, thanks nicolas!!

**Review by nicolasnoble — APPROVED** [review]

**Pull request merged**